themes/bootstrap3/templates/_ui/components/menu-button.phtml

```diff
@@ -0,0 +1,30 @@
+<?php
+  $wrapperTag = $this->wrapperTag ?? 'li';
+  $wrapperClass = $this->wrapperClass ?? 'li';
```

Comment:
Does it really make sense for the default class to be `li`? For that matter, does it make sense for the default wrapper tag to be `li`? Isn't that kind of specific to our use case; would something like `div` be more generic?

Reply:
I see that you fixed the incorrect default wrapper class, but you didn't change the default tag. Is that intentional? (If so, that's fine, just want to be sure this comment wasn't overlooked before I resolve the thread).